src/updatedb.py

```diff
@@ -150,11 +150,12 @@ def uploadResults(data, branch, revision, date):
                    
         testtype = r[i("ref_data_name")].split()[-1]
         if r[i("build_system_type")] == "taskcluster":
-            #TODO: this is fragile, current platforms as of Jan 26, 2016 we see in taskcluster
             pmap = {"linux64": "Linux64",
                     "linux32": "Linux32",
                     "osx-10-7": "MacOSX64",
                     "gecko-decision": "gecko-decision",
+                    "b2g-device-image": "B2G",
+                    "mulet-linux64": "Mulet",
```

Comment:
hmm, not I actually haven't thought about that. I just saw that TODO issue and want to fix that. Then i query each of the platform and add it into the map. That's it.